src/covid_model_infections/cluster.py

```diff
@@ -10,7 +10,7 @@
 from loguru import logger
 
 
-PROJECT = 'proj_covid_prod'
+PROJECT = 'proj_covid'
```

Comment:
We will want to keep the prod project for the daytime runs (based on my understanding that it has higher priority status). I suppose project would need to be an argument in the cli or something if we were to have this in a branch also used by production.